<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLDemosClass</class>
 <widget class="QMainWindow" name="MLDemosClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MLDemos</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="canvasArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="canvasWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>634</width>
         <height>382</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="canvasControlsWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,2">
       <item>
        <widget class="QComboBox" name="canvasTypeCombo">
         <item>
          <property name="text">
           <string>Standard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Scatterplots</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Parallel Coordinates</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Radial Graph</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Andrews Plots</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="canvasZoomSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_SVG"/>
     <addaction name="actionExportOutput"/>
     <addaction name="actionExportAnimation"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuDisplay_Options">
     <property name="title">
      <string>Toolbar Options</string>
     </property>
     <addaction name="actionShow_Toolbar"/>
     <addaction name="actionSmall_Icons"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionClearModel"/>
    <addaction name="actionClearData"/>
    <addaction name="separator"/>
    <addaction name="menuDisplay_Options"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Plugins</string>
    </property>
    <widget class="QMenu" name="menuInput_Output">
     <property name="title">
      <string>Input / Output</string>
     </property>
    </widget>
    <addaction name="menuInput_Output"/>
    <addaction name="separator"/>
    <addaction name="actionManage_Plugins"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClearData">
   <property name="text">
    <string>Clear Data</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClassifiers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Classification</string>
   </property>
  </action>
  <action name="actionRegression">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Regression</string>
   </property>
  </action>
  <action name="actionClustering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clustering</string>
   </property>
  </action>
  <action name="actionDrawSamples">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Samples</string>
   </property>
  </action>
  <action name="actionProjections">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Projections</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionDisplayOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Options</string>
   </property>
  </action>
  <action name="actionClearModel">
   <property name="text">
    <string>Clear Model</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExportOutput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Output Data</string>
   </property>
   <property name="statusTip">
    <string>Export algorithm output for current samples</string>
   </property>
  </action>
  <action name="actionExportAnimation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Parameter Animation</string>
   </property>
   <property name="statusTip">
    <string>Export image sequence animating selected parameters</string>
   </property>
  </action>
  <action name="actionManage_Plugins">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage Plugins</string>
   </property>
  </action>
  <action name="actionAsd">
   <property name="text">
    <string>none</string>
   </property>
  </action>
  <action name="actionShow_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
  </action>
  <action name="actionSmall_Icons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small Icons</string>
   </property>
  </action>
  <action name="actionExport_SVG">
   <property name="text">
    <string>Vector Image (SVG)</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>Comma Separated Values</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
